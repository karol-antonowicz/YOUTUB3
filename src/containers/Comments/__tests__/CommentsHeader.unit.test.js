import React from 'react';
import {shallow} from 'enzyme';
import {CommentsHeader} from '../CommentsHeader';

describe('CommentsHeader', ()=>{
    test('CommentsHeader renders with no props', () => {
        const wrapper = shallow(<CommentsHeader/>);
        expect(wrapper).toMatchSnapshot();
    });
    test('CommentsHeader renders with props amountCommentsHeader', () => {
        const wrapper = shallow(<CommentsHeader amountCommentsHeader={245454}/>);
            expect(wrapper).toMatchSnapshot();
    });
})