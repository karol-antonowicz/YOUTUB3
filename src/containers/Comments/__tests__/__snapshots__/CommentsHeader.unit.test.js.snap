// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader CommentsHeader renders with no props 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort By
  </Button>
</div>
`;

exports[`CommentsHeader CommentsHeader renders with props amountCommentsHeader 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort By
  </Button>
</div>
`;
